money/currency.py

```diff
@@ -1,1308 +1,1136 @@
-        },
-        Currency.XDR: {
-            'display_name': 'SDR (Special Drawing Right)',
-            'numeric_code': 960,
-            'default_fraction_digits': 0,
-            'sub_unit': 100,
-        },
-        Currency.XFU: {
-            'display_name': 'UIC-Franc',
-            'numeric_code': 958,
-            'default_fraction_digits': 0,
-            'sub_unit': 100,
-        },
-        Currency.XOF: {
-            'display_name': 'CFA Franc BCEAO',
-            'numeric_code': 952,
-            'default_fraction_digits': 0,
-            'sub_unit': 100,
-        },
-        Currency.XPD: {
-            'display_name': 'Palladium',
-            'numeric_code': 964,
-            'default_fraction_digits': 0,
-            'sub_unit': 100,
-        },
-        Currency.XPF: {
-            'display_name': 'CFP Franc',
-            'numeric_code': 953,
-            'default_fraction_digits': 0,
-            'sub_unit': 100,
-        },
-        Currency.XPT: {
-            'display_name': 'Platinum',
-            'numeric_code': 962,
-            'default_fraction_digits': 0,
-            'sub_unit': 100,
-        },
-        Currency.XSU: {
-            'display_name': 'Sucre',
-            'numeric_code': 994,
-            'default_fraction_digits': 0,
-            'sub_unit': 100,
-        },
-        Currency.XTS: {
-            'display_name': 'Codes specifically reserved for testing purposes',
-            'numeric_code': 963,
-            'default_fraction_digits': 0,
-            'sub_unit': 100,
-        },
-        Currency.XUA: {
-            'display_name': 'ADB Unit of Account',
-            'numeric_code': 965,
-            'default_fraction_digits': 0,
-            'sub_unit': 100,
-        },
-        Currency.YER: {
-            'display_name': 'Yemeni Rial',
-            'numeric_code': 886,
-            'default_fraction_digits': 2,
-            'sub_unit': 100,
-        },
-        Currency.ZAR: {
-            'display_name': 'Rand',
-            'numeric_code': 710,
-            'default_fraction_digits': 2,
-            'sub_unit': 100,
-        },
-        Currency.ZMW: {
-            'display_name': 'Zambian Kwacha',
-            'numeric_code': 967,
-            'default_fraction_digits': 2,
-            'sub_unit': 100,
-        },
-        Currency.ZWL: {
-            'display_name': 'Zimbabwe Dollar',
-            'numeric_code': 932,
-            'default_fraction_digits': 2,
-            'sub_unit': 100,
-        }
-    }
-    """Data about currencies.
-
-    Taken from https://github.com/sebastianbergmann/money
 
+class CurrencyBase(Enum):
+    """
+    Base class for `Currency` Enum defining fields and methods for each Enum entry
     """
+    # pylint: disable=invalid-name
+    # defining fields below is useful for type assist
+    display_name: str
+    numeric_code: int
+    default_fraction_digits: int
+    sub_unit: int
+    # pylint: enable=invalid-name
 
-    @classmethod
```

Comment:
Best is to consider  it as deletion of entire `CurrencyHelper` class and making new Enum. The currency properties are now directly associated with the Enum